rclpy/rclpy/node.py

```diff
@@ -27,14 +27,16 @@ def __init__(self, handle):
     def create_publisher(self, msg_type, topic, qos_profile):
```

Comment:
would it be appropriate to set the profile to the default profile here, so that the [talker example](https://github.com/ros2/examples/blob/1df40225eb0c0e6e8d1a201093fe90b0a247c898/rclpy_examples/talker_py.py#L33) can simplified further? or is it complicated because of default vs system default